mllib/src/main/scala/org/apache/spark/ml/feature/QuantileDiscretizer.scala

```diff
@@ -49,6 +49,19 @@ private[feature] trait QuantileDiscretizerBase extends Params
 
   /** @group getParam */
   def getNumBuckets: Int = getOrDefault(numBuckets)
+
+  /**
+   * Relative error (see approxQuantile documentation for description).  Must be >= 0.
+   * default: 0.01
```

Comment:
This still says `default: 0.01`. I think we should say something like `By default, the relative error will be in the range [0.001, 0.01]. The exact value is computed based on the number of buckets, decreasing as the number of buckets increases.`